src/commands/new.ts

```diff
@@ -72,23 +72,24 @@ export default {
       filesystem.copy(path(boilerplatePath, ".gitignore"), gitPath)
     }
 
-    // expo-specific changes
+    // Update package.json. Having a "prepare" script in package.json
+    // messes up expo-cli init above (it fails because npm-run-all hasn't been
+    // installed yet), so we're adding it here; we also need to merge in our
+    // extra expo stuff.
+    let packageJson = filesystem.read("package.json", "json")
+    packageJson.scripts.prepare = "npm-run-all patch hack:*"
```

Comment:
Sorry to be nitpicky, but I think for future maintainability I'd prefer this to be in a separate `package.rncli.json` (or similar) so we know where this script line is coming from more intuitively.

I'm going to be pairing with a dev in a bit, so we might do that for you if that's OK.